# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/dart/hotfix-no-symbols.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- dart/sdk/runtime/tools/dart_profiler_symbols.py.vanilla	2024-06-17 14:47:18.751093103 +0200
+++ dart/sdk/runtime/tools/dart_profiler_symbols.py	2024-06-17 14:53:34.048576473 +0200
@@ -75,8 +75,8 @@
         continue
     symbols.append(Symbol(offset, size, name.encode('utf-8')))
 
-if len(symbols) == 0:
-    raise Exception(binary + " has no symbols")
+#if len(symbols) == 0:
+#    raise Exception(binary + " has no symbols")
 
 stream = open(output, "wb")
 stream.write(struct.pack("I", len(symbols)))
